### PR TITLE
add missing .length

### PR DESCRIPTION
**Pull request by mvayngrib**

### CODE REVIEW

**Comment by nolanlawson** [comment]
Thanks for the PR - I just need to fix Saucelabs/Travis integration, and then I should be able to verify that this passes our tests. Much appreciated!

File: index.js [hunk 1]
```diff
@@ -105,7 +105,7 @@ LDIterator.prototype._next = function (callback) {
 
         self.initCompleted = true;
         var i = -1;
-        while (++i < self.onInitCompleteListeners) {
+        while (++i < self.onInitCompleteListeners.length) {
```
Comment (nolanlawson) [inline index.js]: How was this even passing before? Guessing we never even hit this line of code.

Comment (mvayngrib) [inline index.js]: yea, I somehow managed to hit it in my project, need to define a breaking test

**Comment by nolanlawson** [comment]
Closing in favor of https://github.com/No9/localstorage-down/pull/71 due to our newly-unborked Travis situation.

File: index.js (hunk 1 above)
Comment (nolanlawson) [inline index.js]: once we get coverage tests in (#72), this should be a lot easier to catch and measure. it was sloppy of us not to test this in the past.

**Pull request closed without merging**